lib/chef/cookbook/gem_installer.rb

```diff
@@ -66,8 +66,13 @@ def install
                 tf.close
                 Chef::Log.trace("generated Gemfile contents:")
                 Chef::Log.trace(IO.read(tf.path))
-                so = shell_out!("bundle install", cwd: dir, env: { "PATH" => path_with_prepended_ruby_bin })
-                Chef::Log.info(so.stdout)
+                # Skip installation only if Chef::Config[:skip_gem_metadata_installation] option is true
+                unless Chef::Config[:skip_gem_metadata_installation]
+                  # Install gems from local cache if Chef::Config[:gem_installer_bundler_options] is set to :local
+                  cmd = Chef::Config[:gem_installer_bundler_options] == :local ? "bundle install --local" : "bundle install"
```

Comment:
can we do  `cmd = [ "bundle", "install", Chef::Config[:gem_installer_bundler_options] ]` here?

Reply:
Added cmd = %Q{bundle install #{Chef::Config[:gem_installer_bundler_options]}} here.